boilerplate/_lib/container-make

```diff
@@ -14,7 +14,7 @@ CONTAINER_ENGINE=$(command -v podman || command -v docker)
 
 # First set up a detached container with the repo mounted.
 banner "Starting the container"
-container_id=$($CONTAINER_ENGINE run -d -v "$REPO_ROOT":"$REPO_ROOT" $IMAGE_PULL_PATH tail -f /dev/null)
+container_id=$($CONTAINER_ENGINE run --userns keep-id -d -v "$REPO_ROOT":"$REPO_ROOT":Z $IMAGE_PULL_PATH tail -f /dev/null)
```

Comment:
This should be excluded from the MR and managed via `make boilerplate-update`.
Which we'll be likely need to do in the coming week or two.

Reply:
https://github.com/openshift/managed-upgrade-operator/pull/219/commits/16baf6e27566dac0d73a8b3c85eadeda77025f81